src/app/services/mail/mail.service.ts

```diff
@@ -497,7 +502,7 @@ export class MailService {
         },
       }
 
-      return this.#sendNodeMail(mail, { mailId: 'bounce' }).mapErr((error) => {
+      return this.#sendNodeMail(mail, { mailId: 'warning' }).mapErr((error) => {
```

Comment:
Should this be changed? This `mailId` is for `sendBounceNotification`.

I'm guessing that this might be a typo.